synapse/cortex.py

```diff
@@ -1493,14 +1475,58 @@ async def initServicePassive(self):
         for layer in self.layers.values():
             await layer.initLayerPassive()
 
+        await self.finiStormPool()
+
+    async def initStormPool(self):
+
+        byts = self.slab.get(b'storm:pool', db='cell:conf')
+        if byts is None:
+            return
+
+        url, opts = s_msgpack.un(byts)
+
+        self.stormpoolurl = url
+        self.stormpoolopts = opts
+
+        self.stormpool = await s_telepath.open(url)
+
+        # make this one a fini weakref vs the fini() handler
+        self.onfini(self.stormpool)
+
+    async def finiStormPool(self):
+
         if self.stormpool is not None:
             await self.stormpool.fini()
             self.stormpool = None
 
-    async def initQueryMirror(self):
-        if self.stormpoolurl is not None:
-            self.stormpool = await s_telepath.open(self.stormpoolurl)
-            self.onfini(self.stormpool.fini)
+    async def setStormPool(self, url, opts):
+        return await self._push('set:storm:pool', url, opts)
```

Comment:
```suggestion
        return await self._push('storm:pool:set', url, opts)
```
Looks like it's `storm:pool:set` below (L1511) which makes more sense.

Reply:
Whew! Good catch, thanks!